crypto/fipsmodule/rand/rand.c

```diff
@@ -12,6 +12,8 @@
  * OF CONTRACT, NEGLIGENCE OR OTHER TORTIOUS ACTION, ARISING OUT OF OR IN
  * CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE. */
 
+// Ensure we can't call OPENSSL_malloc.
+#define _BORINGSSL_PROHIBIT_OPENSSL_MALLOC
```

Comment:
should we also define this in err.c and other places the system allocator is used?

Reply:
Yeah, it's already defined in [those](https://github.com/aws/aws-lc/blob/02e61afe8c9d6cd68ed3fa664ffd0a55312ad35b/crypto/err/err.c#L110) other [places](https://github.com/aws/aws-lc/blob/e6e201b9886021eb54f32e3030c438c8e7f1570d/crypto/thread_pthread.c#L16).